media/css/firefox/personal.less

```diff
@@ -0,0 +1,333 @@
+        padding-top: 40px;
+        font-family: 'FiraSans Light';
+        .font-size(4.5rem);
+        font-weight: bold;
+        line-height: 1;
+        text-transform: uppercase;
+        letter-spacing: -0.035em;
+        text-shadow: none;
+
+        @media only screen and (max-width: @breakDesktop) {
+            .font-size(3rem);
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            font-family: 'Fira Sans';
+            .font-size(2.5rem);
+            letter-spacing: -1px;
+        }
+
+        span {
+            display: block;
+            color: #fff;
+            margin-left: 28%;
+
+            @media only screen and (max-width: @breakMobileLandscape) {
+                margin-left: 0;
+            }
+        }
+    }
+    h1:before {
+        display: block;
+        background: transparent url(/media/img/firefox/personal/logo.png) 0 0 no-repeat;
+        background-size: 148px;
+        margin: 0 auto @baseLine;
+        width: 148px;
+        height: 148px;
+        content: '';
+        .at2x('/media/img/firefox/personal/logo.png', 295, 295);
+    }
+    .download-button.download-button-simple {
+        /* copied from buttons.less :: .button-flat - with amendments */
+        background: #6fbe4a;
+        color: #fff;
+        padding: 12px (@baseLine * 2);
+        margin-bottom: 0;
+        border: none;
+        cursor: pointer;
+        max-width: 300px;
+        max-height: 68px;
+        font-family: 'FiraSans Light';
+        font-weight: bold;
+        .font-size(1.5rem);
+        text-align: center;
+        .inline-block();
+        .border-radius(10px);
+        .transition();
+
+        @media only screen and (max-width: @breakMobileLandscape) {
+            padding: 6px @baseLine;
+        }
+
+        @media only screen and (max-width: @breakMobile) {
+            min-width: 260px;
+            .font-size(1rem);
+        }
+
+        &:hover {
+            background-color: #7dc14c;
+        }
+        .download-link {
+            box-shadow: none;
+            background-color: inherit;
+            background-image: none;
+            font-family: inherit;
+            font-weight: bold;
+            text-shadow: none;
+            /* for IE */
+            filter: none;
+        }
+    }
+    .download-content {
+        text-transform: uppercase;
+    }
+}
+main {
+    margin: 0 auto;
+    padding: 60px 0 150px 0;
+    width: 100%;
+    max-width: 1400px;
+    text-align: center;
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        padding: 20px 0;
+    }
+}
+.mission-icons-sprite(@posx, @posy) {
+    display: block;
+    background-image: url(/media/img/firefox/personal/mission-icons-sprite.png);
+    background-position: @posx @posy;
+    background-size: 175px;
+    background-repeat: no-repeat;
+    margin-top: @baseLine;
+    height: 190px;
+    content: '';
+    .at2x('/media/img/firefox/personal/mission-icons-sprite.png', '50%', 0);
```

Comment:
Ah so that is why, ok. I saw that it takes the path, width, height and also, whether I should specify the actual size of the high-res or the scaled down size. From your comments above it seems the latter though.

Thanks for the info @alexgibson